tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -58,37 +66,150 @@ func TestServiceEntry(t *testing.T) {
 			url:               "http://bing.com",
 			shouldBeReachable: false,
 		},
-		// See issue https://github.com/istio/istio/issues/7869
-		//{
-		//	name:              "REACHABLE_wikipedia.org_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://www.wikipedia.org",
-		//	shouldBeReachable: true,
-		//},
-		//{
-		//	name:              "UNREACHABLE_google.com_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://google.com",
-		//	shouldBeReachable: false,
-		//},
-		{
-			name:              "REACHABLE_en.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
-			url:               "https://en.wikipedia.org/wiki/Main_Page",
+		{
+			name:              "UNREACHABLE_bing.com_over_bing_wildcard_443",
```

Comment:
UNREACHABLE_www_bing.com_443_over_bing_wildcard_80

Reply:
come to think of it, this seems redundant since we setup service entry for port 80 and not 443. So this test will always pass. not much use

Reply:
the purpose is to validate we indeed only allow 80 but not 443 when service entry is only for 80 not 443.  There was some concern earlier that our service entry allow for stuff that should not be allowed.